acme_tiny.py

```diff
@@ -165,7 +165,7 @@ def _send_signed_request(url, payload):
     return """-----BEGIN CERTIFICATE-----\n{0}\n-----END CERTIFICATE-----\n""".format(
         "\n".join(textwrap.wrap(base64.b64encode(result).decode('utf8'), 64)))
 
-def main(argv):
+def main(argv=None):
```

Comment:
Because the entrypoints system of setuptools expects a callable with no arguments